doc/nrf/known_issues.rst

```diff
@@ -876,6 +877,15 @@ NCSDK-7982: partition manager: Incorrect partition size linkage from name confli
 DFU and FOTA
 ============
 
+.. rst-class:: v1-7-0
+
+NCSDK-11308: Powering off device immediately after serial recovery of the nRF53 network core firmware results in broken firmware.
+  The network core will not be able to boot if the device is powered off too soon after completing a serial recovery update procedure of the network core firmware.
+  This because the firmware is being copied from shared RAM to network core flash **after** mcumgr indicates that the serial recovery procedure has completed.
```

Comment:
```suggestion
  This is because the firmware is being copied from shared RAM to network core flash *after* mcumgr indicates that the serial recovery procedure has completed.
```